server/sonar-server/src/main/java/org/sonar/server/authentication/event/AuthenticationEventImpl.java

```diff
@@ -34,7 +34,7 @@
   private static final int FLOOD_THRESHOLD = 128;
 
   @Override
-  public void login(HttpServletRequest request, @Nullable String login, Source source) {
+  public void loginSuccess(HttpServletRequest request, @Nullable String login, Source source) {
     requireNonNull(request, "request can't be null");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "request can't be null" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1192)